gtsam/linear/NoiseModel.h

```diff
@@ -727,6 +727,7 @@ namespace gtsam {
         Null(const ReweightScheme reweight = Block) : Base(reweight) {}
         virtual ~Null() {}
```

Comment:
Why some of the destructors are virtual and some not?

Reply:
Oops I missed that. Fixed.